Magento_Contact/styles/_module.scss

```diff
@@ -0,0 +1,3 @@
+@import 'modules/button';
+@import 'modules/form';
+@import 'modules/input';
```

Comment:
You do not have to import styles to this module, style are already imported from alpaca component repo in main `styles.scss` file. To delete